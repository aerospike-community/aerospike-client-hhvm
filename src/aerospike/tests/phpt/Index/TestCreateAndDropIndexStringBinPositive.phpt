--TEST--
createIndex and dropIndex- correct arguments for string index

--FILE--
<?php
include dirname(__FILE__)."/../../astestframework/astest-phpt-loader.inc";
aerospike_phpt_runtest("Index", "testCreateAndDropIndexStringBinPositive");
--EXPECT--
OK

