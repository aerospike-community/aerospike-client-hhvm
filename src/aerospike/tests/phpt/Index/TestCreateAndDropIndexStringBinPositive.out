Fatal error: Call to undefined method Aerospike::createIndex() in /home/brusewayne/sandbox/hhvm/aerospike-client-hhvm/src/aerospike/tests/Index.inc on line 184